meta/saiserialize.c

```diff
@@ -803,8 +808,20 @@ int sai_serialize_ip_address(
 
         default:
 
-            SAI_META_LOG_WARN("invalid ip address family: %d", ip_address->addr_family);
```

Comment:
why this is removed? make no sense to serialize IP address if it's not ip4 or ip6, since you dont know what you want to serialize

Reply:
See this corner case.
SAI implementation code may want to serialize an attribute passed by NOS. The attribute is _Inout_ parameter, there is a chance that only attribute.id is set by NOS, but the attribute.value(which contains ipaddress) is random data. In this case, the whole serialize of attribute will be failed if the random addr_family is causing error.
So that I suggest do the serialize in best effort way: do serialize according with the data structure, if failed, serialize the data as array of uint8.

Reply:
random data is invalid, should be set to zero

Reply:
Even if addr_family is 0, it's still an invalid value when serializing the ip address, that's because only 4 and 6 are valid values.